### DISCUSSION TITLE
How to correctly use createMetadataMap for array fields.

### DISCUSSION THREAD

**Discussion by t0xa** [nartc/mapper#278] [Q&A] [answered]
Please consider following interface:

```typescript
export interface SimpleBar {
  bar: string;
  foo: string[];
}
```

What is the right to create `createMetadataMap` for it? `createMetadataMap<SimpleBar>('SimpleBar', { bar: String, foo: Array });`?

**Comment by nartc** [comment, marked as answer]
It will be just `String`. I guess this is missing from the docs, I’ll add it
